utils/src/test/kotlin/OrtAuthenticatorTest.kt

```diff
@@ -66,18 +66,26 @@ class OrtAuthenticatorTest : WordSpec({
             }
         }
 
-        "prefer machine-specific entries over the default" {
+        "recognize a default entry if it is the only entry" {
+            val authentication = getNetrcAuthentication("""
+                default login foo password bar
+            """.trimIndent(), "gitlab.com")
+
+            authentication shouldNotBeNull {
+                userName shouldBe "foo"
+                password shouldBe "bar".toCharArray()
+            }
+        }
+
+        "disregard a default entry that comes before machine entries" {
             val authentication = getNetrcAuthentication("""
                 default login foo password bar
                 machine github.com
                 login git
                 password hub
-            """.trimIndent(), "github.com")
+            """.trimIndent(), "gitlab.com")
```

Comment:
Can you add an additional test case that checks if a host can be found if the `.netrc` format is ok?
As this was the original purpose of this test.

Reply:
Have you also checked the other existing tests that I left untouched? Can you state exactly what additional case you'd like to see covered?

Reply:
I checked the other tests now. The case I'd like to see covered is this `netrc`:
```
machine host.com login user password pass
default login foo password bar
```

Reply:
Your test name is `"disregard a default entry that comes before machine entries"`
But your test checks if `gitlab.com` can be found, while it should actually check for `github.com` to check the if the default entry is disregarded.

Reply:
My previous comment is invalid, as you're right, there is already a test that covers that scenario.

However:
I'd suggest to add a test case that actually tests the behaviour that is mentioned in the `netrc` documentation:
Something like that:
```kt
"prioritise a default entry if it comes before the actual host" {
    val authentication = getNetrcAuthentication("""
        default login foo password bar
        machine github.com
        login git
        password hub
    """.trimIndent(), "github.com")

    authentication shouldNotBeNull {
        userName shouldBe "foo"
        password shouldBe "bar"
    }
}
```